### PR TITLE
LPD-25781 Fix key

### PR DESCRIPTION
**Pull request by chestofwonders**
Bug: https://liferay.atlassian.net/browse/LPD-25781

image-20240510-073712

Key `no-filter-sources-available.-create-a-picklist-or-a-vocabulary-for-this-type-of-filter` not displayed properly

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/filters/modal_content/selection_filter/SelectionFilter.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ function Body({
 		return (
 			<ClayAlert displayType="info" title="Info">
 				{Liferay.Language.get(
-					'no-filter-sources-are-available.-create-a-picklist-or-a-vocabulary-for-this-type-of-filter'
+					'no-filter-sources-available.-create-a-picklist-or-a-vocabulary-for-this-type-of-filter'
```
Comment (juanjofgliferay) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/filters/modal_content/selection_filter/SelectionFilter.tsx:79]: Wouldn't be better to update the key in the `Language.properties` file?

There are more sentences like this, x.e  `no-frontend-data-set-filter-client-extensions-are-available.-...`

**Review by juanjofgliferay — APPROVED** [review]

**Review by markocikos — APPROVED** [review]
@chestofwonders LGTM! :sparkles:

**Comment by chestofwonders** [comment]
ci:forward

**Comment by chestofwonders** [comment]
ci:forward

**Comment by markocikos** [comment]
ci:forward

**Pull request closed without merging**